docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+# Copy this into a file called `docker-compose.yml`
+version: "3.2"
+
+    services:
+      todo_app_prod:
+        container_name: production
+        image: todo-app:prod
+        build: 
+          context: .
+          dockerfile: Dockerfile
+        volumes: 
+          - type: volume
+            source: ./todo_app
+            target: /todo_app/todo_app
+        ports:
+            - 5000:5000
+        env_file:
+            - .env
+
+      todo_app_dev:
+        container_name: Development
+        image: todo-app:dev
+        build: 
+          context: .
+          dockerfile: Dockerfile
+        volumes: 
+          - type: volume
+            source: ./todo_app
+            target: /todo_app/todo_app
+        ports:
+          - 5001:5000
+        env_file:
+          - .env
+          
+      todo_app_test:
+        container_name: tests
+        image: todo-app:test
+        build: 
+          context: .
+          dockerfile: Dockerfile
+        volumes: 
+          - type: volume
+            source: ./tests
+            target: /todo_app/tests
+            env_file:
+              - .env.test
+          
+      todo_app_test_e2e:
```

Comment:
e2e tests need a browser and webdriver to run. Your Dockerfile currently doesn't install those.

Reply:
@hugh-emerson , can you please suggest how to add that in Dockerfile or do you suggest to remove todo_app_test_e2e block from docker-compose.yml?